oci/spec_opts.go

```diff
@@ -190,7 +190,9 @@ func WithEnv(environmentVariables []string) SpecOpts {
 // WithDefaultPathEnv sets the $PATH environment variable to the
 // default PATH defined in this package.
 func WithDefaultPathEnv(_ context.Context, _ Client, _ *containers.Container, s *Spec) error {
-	s.Process.Env = replaceOrAppendEnvValues(s.Process.Env, defaultUnixEnv)
+	if s.Windows == nil {
```

Comment:
Can you add a comment above stating that windows containers always receive a default PATH (basically whats in the PR description)